src/Microsoft.ComponentDetection.Detectors/npm/NpmComponentDetector.cs

```diff
@@ -87,6 +88,12 @@ protected virtual bool ProcessIndividualPackageJTokens(string filePath, ISingleF
             return false;
         }
 
```

Comment:
Since you are changing the graph generated by the detector you need to bump the version of the detector see line 43, it should be bumped to 3.

`public override int Version { get; } = 2;`